framework/actions/report.go

```diff
@@ -110,9 +160,74 @@ func Report(ctx context.Context, inputContext *InputContext, planHref string, pl
 
 	assessmentResults.Results[0].Findings = pkg.NilIfEmpty(&oscalFindings)
 
+	// If inventory items were created then add to result
+	if len(invItemMap) > 0 {
+		invItems := make([]oscalTypes.InventoryItem, 0, len(invItemMap))
+
+		for _, invItem := range invItemMap {
+			invItems = append(invItems, invItem)
+		}
+
+		localDefs := oscalTypes.LocalDefinitions{
+			InventoryItems: &invItems,
+		}
+		assessmentResults.Results[0].LocalDefinitions = &localDefs
+	}
+
+	// If resources were created then add to result
+	if len(resourceItemMap) > 0 {
+		resources := make([]oscalTypes.Resource, 0, len(resourceItemMap))
+		for _, r := range resourceItemMap {
+			resources = append(resources, r)
+		}
+		assessmentResults.BackMatter.Resources = &resources
```

Comment:
When testing locally, this paniced because `BackMatter` is nil. Just needs that field initialized. Updated to this below and everything worked well:

```bash
if len(resourceItemMap) > 0 {
		backmatter := oscalTypes.BackMatter{}
		resources := make([]oscalTypes.Resource, 0, len(resourceItemMap))
		for _, r := range resourceItemMap {
			resources = append(resources, r)
		}
		backmatter.Resources = &resources
		assessmentResults.BackMatter = &backmatter
	}
```

Reply:
@jpower432 thank you for catching this, I will fix.